airflow/bin/cli.py

```diff
@@ -1063,3 +1063,23 @@ def _add_subcommand(cls, subparsers, sub):
 def get_parser():
     """Calls static method inside factory which creates argument parser"""
     return CLIFactory.get_parser()
+
+
+def exec_airflow_command(command_to_exec: List[str]):
```

Comment:
We should change the StandardTaskRunner to use this new method too please

Reply:
https://github.com/apache/airflow/blob/master/airflow/task/task_runner/standard_task_runner.py#L71-L84
This will not be easy because StandardTaskRunner also sets the title of the process. For me it is a slightly different case.